### ISSUE TITLE
doesn't really scale

### ISSUE THREAD

**Issue by jusleg** [almiche/BackAtCha#5]
🤔

**Comment by almiche** [comment]
Agreed this is an issue we should think about addressing in our next OKR meeting

**Comment by jusleg** [comment]
Sure, what's OKR?

**Comment by almiche** [comment]
I'll send you a calendar invite for it

**Comment by jusleg** [comment]
cc @senjai

**Comment by almiche** [comment]
Let's book a quick white boarding session sometime after the pandemic. I have an office space on the outskirts of town we can use. Bring your own marker of course.

**Comment by jusleg** [comment]
> Bring your own marker of course.

Will you provide the paper? I only have a Meisterstuck 164.

**Comment by almiche** [comment]
> I only have a Meisterstuck 164.

![image](https://user-images.githubusercontent.com/14715156/78614355-5a811000-783c-11ea-8879-829ac10bcef3.png)
According to this spec diagram that pen might not fit our requirements. Would you be able to source a 149 possibly ? The increased surface area of the nib mated with the heavier body will provide a much more stable writing experience and cleaner overall result.

**Comment by jusleg** [comment]
I'll bring a muji pen just in case
